deployment/README.md

```diff
@@ -2,29 +2,96 @@
 
 Reactive Interaction Gateway (RIG) uses [Peerage library](https://github.com/mrluc/peerage) to do discovery in distributed mode (production Distillery release).
```

Comment:
Do we want to mention that in this readme? It does not really matter how it works under the hood for the user. I would suggest that this deployment guide is as simple as possible to apply.

Reply:
removed it